dartsim/src/ShapeFeatures.hh

```diff
@@ -148,7 +148,7 @@ class ShapeFeatures :
   public: Identity AttachEllipsoidShape(
       const Identity &_linkID,
       const std::string &_name,
-      const Vector3d _radii,
+      Vector3d _radii,
```

Comment:
I think it was meant to be `const Vector3d &_radii`

Reply:
Done